### PR TITLE
Microsoft.PolicyInsights RP Powershell cmdlets

### PR DESCRIPTION
**Pull request by bulentelmaci**
## Description

PowerShell cmdlets for the public preview version of the new Microsoft.PolicyInsights RP. 

Design review: https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/28

## Checklist

- [X] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [X] The title of the PR is clear and informative
- [X] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [X] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [X] If applicable, the changes made in the PR have proper test coverage
- [X] For public API changes to cmdlets:
    - [X] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [X] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by MiYanni** [comment]
Will take a look at this today.

File: src/ResourceManager/PolicyInsights/AzureRM.PolicyInsights.psd1 [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+DotNetFrameworkVersion = '4.5.2'
+
+# Minimum version of the common language runtime (CLR) required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+# CLRVersion = ''
+
+# Processor architecture (None, X86, Amd64) required by this module
+# ProcessorArchitecture = ''
+
+# Modules that must be imported into the global environment prior to importing this module
+RequiredModules = @(@{ModuleName = 'AzureRM.Profile'; ModuleVersion = '4.3.1'; })
```
[the remark this answers is not in the document]
Comment (bulentelmaci) [reply src/ResourceManager/PolicyInsights/AzureRM.PolicyInsights.psd1]: @MiYanni Will change it

**Comment by MiYanni** [comment]
On-demand: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/powershell-demand/
Signed: https://azuresdkci.westus2.cloudapp.azure.com/job/ps-sign/388/

**Comment by bulentelmaci** [comment]
@miyanni Restarted the signed build after it failed with what seemed like a transient issue: https://azuresdkci.westus2.cloudapp.azure.com/job/ps-sign/390

**Comment by bulentelmaci** [comment]
@MiYanni New signed build: https://azuresdkci.westus2.cloudapp.azure.com/job/ps-sign/392/

**Comment by bulentelmaci** [comment]
@cormacpayne @MiYanni The MSI generated by the signed build above did not install my module. What other changes I need to do to be part of AzureRM (both in MSI and in PS gallery)?

**Comment by bulentelmaci** [comment]
New signed build: https://azuresdkci.westus2.cloudapp.azure.com/job/ps-sign/398

**Comment by bulentelmaci** [comment]
New signed build: https://azuresdkci.westus2.cloudapp.azure.com/job/ps-sign/399

**Comment by bulentelmaci** [comment]
@MiYanni @cormacpayne https://azuresdkci.westus2.cloudapp.azure.com/job/ps-sign/400 failed as well. If I'm reading the logs right, for some reason, my module was not built, and/or nuget was not published, which caused AzureRM build to fail, hence wix is failing as well. Please take a look at my latest revision and let me know what I'm missing.

**Comment by bulentelmaci** [comment]
Successful signed job: https://azuresdkci.westus2.cloudapp.azure.com/job/ps-sign/407

**Pull request merged**